src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -164,10 +164,13 @@ public void execute(Boolean arg) {
                      highlighters_.remove(webLinkHighlighter());
                   }
             }});
+            if (fileType.isR())
+               highlighters_.add(issueHighlighter());
```

Comment:
only adding the issue highlighter for `.R` files. Do we have a way to limit to `test-*` files ?

Reply:
I don't think so. I think we'd have to tag our AceEditor instances with the file path they're associated with (if any) and then provide accessors for that.

Reply:
That's not really necessary. The regex does look behind for `"test_that"`. it's probably enough.